app/models/penalty_decay_period_submission_rule.rb

```diff
@@ -71,7 +71,7 @@ def calculate_penalty(overtime_hours)
       if overtime_hours / interval < 1
         total_penalty = total_penalty + deduction
       else
-        total_penalty = total_penalty + (overtime_hours/interval).to_i * deduction
+        total_penalty = total_penalty + (overtime_hours/interval).to_f * deduction
```

Comment:
Line is too long. [82/80]<br>Use self-assignment shorthand `+=`.<br>Surrounding space missing for operator `/`.